datatypes/papas/DefinitionsCollections.h

```diff
@@ -0,0 +1,42 @@
+//
+//  DefinitionsCollections.h
+//  papas
+//
+//  Created by Alice Robson on 26/05/16.
+//
+//
+
+#ifndef DefinitionsCollections_h
+#define DefinitionsCollections_h
+
+#include "Id.h"
+//#include <map>
+#include <unordered_map>
+#include <vector>
+namespace papas {
+class Cluster;
```

Comment:
isn't the indentation wrong? the code should be indented from a further step in each new nested context. 
Because of that I misread and thought at first that the collections where not in the papas namespace.